app/controllers/users_controller.rb

```diff
@@ -1,6 +1,7 @@
 class UsersController < ApplicationController
   before_action :set_user, only: [:show, :edit, :update, :destroy]
   before_action :authenticate_user!
+  # before_action :require_admin
```

Comment:
I like where you were going with this. Is there a reason not to do it? If so, remove the comment. Is there something that runs `require_admin` right now? I can still access users if I put the url in manually when logged in as non-admin user so I don't think your protection code is running.

Reply:
add require_admin back and include admin test